src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -1,36 +1,23 @@
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
+import { EditorState } from 'draft-js';
+import { mount, ReactWrapper } from 'enzyme';
 import ReplyField, { Props } from '../ReplyField';
 
 describe('components/Popups/ReplyField', () => {
     const defaults: Props = {
-        className: 'ba-Popup-text',
-        disabled: false,
-        defaultValue: '',
+        className: 'ba-Popup-field',
+        editorState: EditorState.createEmpty(),
+        isDisabled: false,
         onChange: jest.fn(),
-        onClick: jest.fn(),
     };
 
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ReplyField {...defaults} {...props} />);
-
-    describe('event handlers', () => {
-        test('should call onChange event', () => {
-            const wrapper = getWrapper();
-            const textarea = wrapper.find('textarea');
-
-            textarea.simulate('change', 'test');
-            textarea.simulate('click', 'test');
-
-            expect(defaults.onChange).toHaveBeenCalledWith('test');
-            expect(defaults.onClick).toHaveBeenCalledWith('test');
-        });
-    });
+    const getWrapper = (props = {}): ReactWrapper => mount(<ReplyField {...defaults} {...props} />);
```

Comment:
The react version is not compatible to use shallow rendering for function component with hook